paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -196,14 +196,14 @@ def get_mesos_utilization_error(self,
                     current_instances, expected_instances, MISSING_SLAVE_PANIC_THRESHOLD)
                 raise ClusterAutoscalingError(error_message)
 
-        pool_utilization_dict = get_resource_utilization_by_grouping(
-            lambda slave: slave['attributes']['pool'],
+        region_pool_utilization_dict = get_resource_utilization_by_grouping(
+            lambda slave: (slave['attributes']['pool'], slave['attributes']['datacenter'],),
```

Comment:
This 'datacenter' word really shouldn't be in here, but this is slightly better than what we do now.

I still think the long term solution is to "just" look at the slaves that are in the resource itself.